docs/Changelog.md

```diff
@@ -34,7 +34,8 @@ Nussknacker versions
 * [#2684](https://github.com/TouK/nussknacker/pull/2684) Handled 'Restarting' state in Embedded DeploymentManager when the embedded scenario is failing
 * [#2686](https://github.com/TouK/nussknacker/pull/2686) Rename `ServiceWithStaticParameters` to `EagerServiceWithStaticParameters` to avoid confusion about lazy and eager parameters used by default
 * [#2695](https://github.com/TouK/nussknacker/pull/2695) Replaced `nodeId` with `NodeComponentInfo` in `NuExceptionInfo`
-
+* [#2712](https://github.com/TouK/nussknacker/pull/2712) Embedded DeploymentManager now starts in During Deploy state before it changes to Running
```

Comment:
Please remove this changelog entry, embedded DM is still an undocumented feature - let's not mess changelog, it can be confusing for users.

Reply:
Good question! looks like @mproch leaked this hidden feature ;-) But despite this, our changelog should be written from perspective of user who has knowledge about what was in  version N-1 and want to know what's new is in version N. Would he/she be interested in fact that "Embedded DeploymentManager" was implemented in a few steps by us? I propose to just add this PR link to entry about "Embedded DeploymentManager" instead of adding new entry